terraform/projects/app-ecs-albs/main.tf

```diff
@@ -129,19 +129,127 @@ resource "aws_lb_listener" "monitoring_external_listener" {
   }
 }
 
+resource "aws_lb" "alertmanager_external_alb" {
+  name               = "${var.stack_name}-alertmanager"
+  internal           = true
+  load_balancer_type = "application"
+  security_groups    = ["${data.terraform_remote_state.infra_security_groups.aletmanager_external_sg_id}"]
+
+  subnets = [
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 0)}",
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 1)}",
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 2)}",
+  ]
+
+  tags = "${merge(
+    local.default_tags,
+    var.additional_tags,
+    map("Stackname", "${var.stack_name}"),
+    map("Name", "${var.stack_name}-ecs-monitoring")
```

Comment:
Is this the best Name for the ALB? Surely `"${var.stack_name}-alertmanager"` would be better?

Reply:
map("Name", "${var.stack_name}-internal-alb")

I have renamed it to this, since we have multiple services on the alb that exposed internally. When we go multi-AZ we will need to add the the count to the alb name or AZ association.